docs/custom-styles.html

```diff
@@ -65,3 +65,20 @@
         margin-left: 0;
     }
 </style>
+<script type="text/javascript">
```

Comment:
We should not put a global script to execute for progress bar. This also I don't think will work in skin demo pages.

You should add it in `main.js` if needed. 

You can also just have a static one at different values.

Reply:
oh so I could put this script in the `docs/_includes/common/progress-bar.html` file? 

I tried getting it to work in the main.js file but it wasn't working at all...

Reply:
Don't put it in there

It needs to go in `docs/src/js/base/main.js`

Reply:
okay so I put it in `docs/src/js/base/main.js` and it doesn't work. Pasting it in the browser does work, however.

Maybe there's an issue because the docs are server rendered so `window` isn't accessible?